spec/lib/services/profiles-api-service-spec.js

```diff
@@ -138,6 +152,24 @@ describe("Http.Services.Api.Profiles", function () {
             });
         });
 
+        it("setLookup should set lookup when macs and ips exists in query", function() {
+            this.sandbox.stub(lookupService, 'setIpAddress').resolves();
+
+            return profileApiService.setLookup(profileReq)
+            .then(function(result) {
+                expect(lookupService.setIpAddress).to.be.calledTwice;
```

Comment:
It's not enough to only checked the callCount, suggest adding some assertion logic for callArgs, for example:
```
expect(lookupService.setIpAddress.firstCall).to.be.calledWith('ip1', 'mac1');
expect(lookupService.setIpAddress.secondCall).to.be.calledWith('ip2', 'mac2');
```